matlab_simulation/code v1.0/ME597-5-ExtendedKalmanFilter/omniRobotFrame.m

```diff
@@ -0,0 +1,11 @@
+function [Xideal,X,Y,V]=omniRobotFrame(r,l,W,Xinit,dt,e)
+
```

Comment:
W is vector of the wheels velocity W1 ,W2, W3
V is final velocity of the robot.